lib/osf-components/addon/components/file-actions-menu/template.hbs

```diff
@@ -60,6 +60,16 @@
             </dd.trigger>
         </SharingIcons::Dropdown>
         {{#if @manager}}
+            <OsfLink
+                data-test-metadata-link
+                data-analytics-name='File metadata link'
+                local-class='metadata-link'
+                @href='#'
```

Comment:
This should go to `<file_view_link>/metadata`. I forget exactly what variable is holding the view link. Probably `{{@item.links.html}}` will do it.

Reply:
Ok, I can remember to add that in the future for metadata links.